### DISCUSSION TITLE
I need producer Details

### DISCUSSION THREAD

**Discussion by Ben-SteveJobs-Hadar** [watsonbox/exportify#163] [Q&A] [answered]
Can Anybody help me just add the producers that are listed to what is ripped from the API?

**Comment by watsonbox** [comment]
If producer information is available in the [Web API](https://developer.spotify.com/documentation/web-api), this is possible. Otherwise I'm afraid it's not.

Could you specify exactly which information from the API you believe is missing?

**Comment by Ben-SteveJobs-Hadar** [comment]
Thank you! Yes the information from the API that I believe is missing is
the "produced by" section as shown in the image below... it is always there
when you tap the credits of a song

[image: Screenshot 2024-11-13 at 12.49.46 PM.png]

thank you!

**Comment by watsonbox** [comment, marked as answer]
Unless you can find it in the link I gave above, I'm afraid that information isn't available in the API.

**Comment by Ben-SteveJobs-Hadar** [comment]
Oh ok, thank you
